pandas/io/excel.py

```diff
@@ -194,7 +198,7 @@
 1  string2      2
 2  string3      3
 
->>> pd.read_excel(open('tmp.xlsx','rb'))
+>>> pd.read_excel(open('tmp.xlsx', 'rb'))
```

Comment:
I'd remove this example, may be you can show one with `sheet_name='Sheet3'` instead?